app/webpacker/styles/blocks.scss

```diff
@@ -24,6 +24,7 @@ $space-between-sections: 3.7em;
 
   > * {
     flex: 1 0;
+    max-width: 100%;
```

Comment:
are you sure about this `max-width: 100%`?
it could have unexpected elsewhere on the site - anywhere that is using a `.blocks` class. It might be safer to create a new class specifically for this purpose.